PyInstaller/utils/hooks/hookutils.py

```diff
@@ -29,7 +29,7 @@
 
 # These extensions represent Python executables and should therefore be
 # ignored when collecting data files.
-PY_IGNORE_EXTENSIONS = set(['.py', '.pyc', '.pyd', '.pyo', '.so', 'dylib'])
+PY_IGNORE_EXTENSIONS = set(['.py', '.pyc', '.pyd', '.pyo', '.so', '.dylib'])
```

Comment:
Could you integrate this patch into your pull request? Then the continuous integration should pass.
```patch
--- a/PyInstaller/utils/hooks/hookutils.py
+++ b/PyInstaller/utils/hooks/hookutils.py
@@ -29,7 +29,8 @@ PY_EXECUTABLE_SUFFIXES = set(['.py', '.pyc', '.pyd', '.pyo', '.so'])
 
 # These extensions represent Python executables and should therefore be
 # ignored when collecting data files.
-PY_IGNORE_EXTENSIONS = set(['.py', '.pyc', '.pyd', '.pyo', '.so', '.dylib'])
+# NOTE: .dylib files are not Python executable and should not be in this list.
+PY_IGNORE_EXTENSIONS = set(['.py', '.pyc', '.pyd', '.pyo', '.so'])
 
 # Some hooks need to save some values. This is the dict that can be used for
 # that.
```